packages/components/rate/src/types.ts

```diff
@@ -30,6 +30,10 @@ export const rateProps = {
     default: (): string[] => [],
   },
   size: String as PropType<FormSize>,
+  color: {
+    type: String,
+    default: '#F8D81A',
```

Comment:
```suggestion
    default: undefined
```
默认直给 undefined 就行，默认情况下还是用 css 去控制。

Reply:
文档那里也一并改下。

Reply:
好的，我在弄。我在琢磨这个 git base。